assets/xml/objects/object_ik.xml

```diff
@@ -11,11 +11,21 @@
         <Animation Name="gIronKnuckleFrontHitAnim" Offset="0x5254" />
         <Animation Name="gIronKnuckleRunAnim" Offset="0x57F4" />
         <Animation Name="gIronKnuckleWalkAnim" Offset="0x6294" />
-        <!-- <Blob Name="object_ik_Blob_006730" Size="0x5A0" Offset="0x6730" /> -->
-        <!-- <Blob Name="object_ik_Blob_007120" Size="0x11A0" Offset="0x7120" /> -->
-        <!-- <Blob Name="object_ik_Blob_008AE0" Size="0x630" Offset="0x8AE0" /> -->
-        <!-- <Blob Name="object_ik_Blob_009290" Size="0x2F0" Offset="0x9290" /> -->
-        <!-- <Blob Name="object_ik_Blob_009740" Size="0x300" Offset="0x9740" /> -->
+        <Array Name="gObjectIkVtx_006730" Count="90" Offset="0x6730" Static="On">
+            <Vtx/>
+        </Array>
+        <Array Name="gObjectIkVtx_007120" Count="282" Offset="0x7120" Static="On">
+            <Vtx/>
+        </Array>
+        <Array Name="gObjectIkVtx_008AE06" Count="99" Offset="0x8AE0" Static="On">
+            <Vtx/>
+        </Array>
+        <Array Name="gObjectIkVtx_009290" Count="47" Offset="0x9290" Static="On">
+            <Vtx/>
+        </Array>
+        <Array Name="gObjectIkVtx_009740" Count="48" Offset="0x9740" Static="On">
+            <Vtx/>
+        </Array>
```

Comment:
You should be able to extract all of them in one go, ZAPD should use any symbols correctly. Also, don't turn static on, there's no point.

Reply:
What do you mean by "one go"? Is that putting them all in one array?

Reply:
Yes, one array.